cla_backend/apps/cla_eventlog/tests/test_find_and_delete_old_cases.py

```diff
@@ -228,3 +238,182 @@ def test_old_and_new_cases_are_deleted_correctly(self):
         self.assertEqual(EligibilityCheck.objects.count(), 1)
         self.assertEqual(DiagnosisTraversal.objects.count(), 1)
         self.assertEqual(PersonalDetails.objects.count(), 1)
+
+    def test_log_is_deleted_when_case_is_viewed_by_digital_justice_user(self):
```

Comment:
All of these test cases are extremely verbose and hard to read. Any way of making them shorter by sharing more common stuff and/or just not passing some things (like are the datetimes actually important for these test cases?)

Reply:
I've changed some of the names. Let me know what you think

Reply:
You've only changed the names. I may have been unclear, but it's the test cases that are too long and hard to read. As I say, if you break some of it down into functions, even if you won't have fewer lines of code overall (but you probably will, cos of reusing the function multiple times) it'll mean you can read what the code does without having to read all of how it does it (you can choose to read a function it calls, or not)

Reply:
Will respond to the other comment